lms/static/js/spec/groups/views/cohorts_spec.js

```diff
@@ -1488,7 +1488,7 @@ define(['backbone', 'jquery', 'edx-ui-toolkit/js/utils/spec-helpers/ajax-helpers
 
                         topicsJson = {
                             course_wide_discussions: {
-                                children: ['Topic_C_1'],
+                                children: [['Topic_C_1', 'entry']],
```

Comment:
@robrap I will add the constants in JS tests also but can you please determine the file in which we can add the constants.